packages/server/src/tests/utilities/api/role.ts

```diff
@@ -27,14 +28,18 @@ export class RoleAPI extends TestAPI {
     })
   }
 
-  destroy = async (roleId: string, expectations?: Expectations) => {
-    return await this._delete(`/api/roles/${roleId}`, {
+  destroy = async (role: Role, expectations?: Expectations) => {
+    return await this._delete(`/api/roles/${role._id}/${role._rev}`, {
       expectations,
     })
   }
 
-  accesssible = async (expectations?: Expectations) => {
+  accessible = async (
+    headers: Record<string, string | string[]>,
```

Comment:
I'm not fond of this. Would prefer to see some sort of `config.withHeaders` helper, so it fits with other stuff that we do and doesn't pollute the API here for people that don't want to set headers.